README.md

```diff
@@ -1,15 +1,16 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2021S1-CS2103T-T12-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S1-CS2103T-T12-2/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-  *
+* tCheck is a desktop application that helps bubble tea store managers manage the staffs’ contact information, store’s inventory and historical sales data. It is optimized for CLI users to update and retrieve the information more efficiently.
```

Comment:
Maybe can write down the full name for CLI (Command Line Interface)? In case some users don't know what CLI stands for.

Reply:
updated, thanks!